extensions-robolectric/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="iokotest.kotest">

```

Comment:
```suggestion
<manifest package="io.kotest">

```

Reply:
I thought that the tests for this extension do not work with 'io.kotest'.So I changed the package appropriately.
https://github.com/kotest/kotest-extensions-robolectric/blob/ed8ac0e158c950db1eb1a6710fa0edb4e43d2de6/extensions-robolectric/src/main/kotlin/io/kotest/extensions/robolectric/ContainedRobolectricRunner.kt#L33
I think I changed the main one by mistake at that time.